### ISSUE TITLE
Script file must provide implementation for class VMS.TPS.Script

### ISSUE THREAD

**Issue by 118070** [VarianAPIs/Varian-Code-Samples#85]
Hi,

I am trying to create my first plug-in script for verifying some parameters on Eclipse photon IMRT plan. The script loads beam calculation log and extract the loss MU factor and Maximum MU, then compare their multiplication with beam meter set. The script works well as .cs file on Eclipse. However it run into some problems when I compile the script as .dll file. I found the script as .dll file works on some of our clinical workstation but some workstation can't with an error message as below. The version of all the Eclipse workstation I tested is 15.6.

"There was a problem while executing the script."
"Script file must provide implementation for class VMS.TPS.Script." 

Any idea about how I can solve it? Thanks.

**Comment by Kiragroh** [comment]
My guess: ESAPI is not installed on all workstations. Look for the Scripting wizard on the not working stations. If the wizard is missing you need to install ESAPI. The installation .exe will be given to you if you ask here: eclipsedeveloper@varian.com

**Comment by 118070** [comment]
Hi Kiragroh,

Thanks for your reply. I have checked the not working station and found all of them have Eclipse Script Wizard. 
I also tested the example scripted provided by Varian but they all work well on the workstation which my script can't work. Because we don't have Visual Studio on our workstation, I compile my script to .dll file on my personal computer with Visual Studio 2019 and follow of step of Eclipse Scripting API Reference Guide (Document ID: P1015247-001-A)
